### ISSUE TITLE
PWA loads a 404 on Safari iOS 13.2

### ISSUE THREAD

**Issue by kyleschmolze** [emergentworks/project-not-911#46]
On Safari, iOS 13.2, I can't seem to successfully install the PWA on my Home Screen. If I navigate to the web app (https://not911.nyc/app/) and then attempt to Share > Add to Home Screen, Safari tries to add a PWA using this URL: https://not911.nyc/app/app/ (which is then a 404).

I **can** add the PWA through Safari from https://not911.nyc/ directly, but then the PWA opens the marketing page, not the web app.

I've put screenshots from my phone for both of these cases below.

Might this be caused by [this line](https://github.com/emergentworks/project-not-911/blob/1badd7bac1dc1c5639e76add6abc4ebfb6fa655c/docs/app/manifest.json#L7) in the manifest.json? Using `./app` may mean "current directory plus `/app`", which would explain the `/app/app` I'm seeing. 

A proposed solution would be to simply change the `./app/` to `/app/`, like this:
```
  "start_url": "/app/",
```
so that the current directory is overwritten. Another solution which may be the safest could be to just hard-code the `start_url`, which is supposedly supported, according to  [WebMDN](https://developer.mozilla.org/en-US/docs/Web/Manifest/start_url). That would look like:
```
  "start_url": "https://not911.nyc/app",
```

This all said, I tried reproducing the bug in development, and I can't 😕 I can't even get my changes in `manifest.json` to affect the URL for the PWA in dev. So I might be way off base here on the solution. If so, sorry for the misdirection 😅 But I am getting the 404 in production. 

Screenshots:

<img src='https://user-images.githubusercontent.com/1200038/93493666-04bb9400-f8c1-11ea-92b0-0c0e5474abad.PNG' width='200px' />
<img src='https://user-images.githubusercontent.com/1200038/93493680-07b68480-f8c1-11ea-8f7e-c3c1061ae7dd.PNG' width='200px' />

**Comment by rossPatton** [comment]
So just to follow-up: I've pushed a bunch of changes and tweaks today, but wasn't able to get to this just yet. It looks like a fairly easy fix if your suggestion pans out. I will try to get to it tomorrow - but if not, you are more than welcome to PR a fix yourself!

**Comment by kyleschmolze** [comment]
Yeah I did try to get it working locally and attempt the fix, but I didn't get any changed behavior from my suggestion. I didn't have the time to keep digging - could be that I was editing the wrong spot, or could be that my dev environment isn't set up propertly, or could be that I just don't fully understand the manifest.json yet!

**Comment by rossPatton** [comment]
unfortunately i think this might still be an issue. i took a long break from this project, but am now working on it again more regularly. i will look into it again soon
